Tasks/PowerShellV2/task.json

```diff
@@ -17,13 +17,18 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 190,
+        "Minor": 191,
         "Patch": 0
     },
     "releaseNotes": "Script task consistency. Added support for macOS and Linux.",
     "minimumAgentVersion": "2.115.0",
     "showEnvironmentVariables": true,
     "groups": [
+        {
+            "name": "preferenceVariables",
+            "displayName": "Preference Variables",
+            "isExpanded": false
+        },
```

Comment:
@bradselw @anatolybolshakov

At the moment UI of the task looks like this: 
![image](https://user-images.githubusercontent.com/14060121/128850150-c9f41010-de13-4eb4-9ded-167b972ec381.png)

I would like to suggest another way of how we can introduce this feature:
1) By default `Preference Variables`  group will be not visible
2) Add "Override preference variables" toggle to `Advanced` group 
3) By default"Override preference variables" toggle will be set to `false`
4) If the customer toggle `Override preference variables` the `Preference Variables` group will be displayed

So the final flow will look like this:
![UI suggestion](https://user-images.githubusercontent.com/14060121/128851228-b1e6ce65-f830-4d61-8fc7-aad74d380790.gif)

Also, if we will keep related logic behind the `Override preference variables` toggle it will allow us to avoid unnecessary injection of `preference variables `with the current default values and save the default behavior of the task.

Reply:
Could it make sense to add new 'Default' value for these new inputs - to be able to set it as current default value considered by Powershell, and set it up by default? 
@alexander-smolyakov let's quickly discuss it with team also

Reply:
I'll just point out that the original authors of this task saw fit to explicitly override PowerShell's default action preference for the `Error` stream (which is `Continue`) to `Stop`, because the script is running in an automated environment and you want errors in the script to fail the task. This PR does the same thing for the `Information` stream: because the script is running in an automated environment, any information should be printed to the console instead of ignored.

Reply:
If we were to toggle preferences based on `Override preference variables`, we would break pipelines that rely on the task's default for `ErrorActionPreference`. As a reminder: the task's default for `ErrorActionPreference` is `Stop`, while PowerShell's default is `Continue`.

Reply:
@bradselw got it, thanks! Could you please add the 'Default' value then for new steams like  `Information` - to keep previous task behavior by default? (default Powershell value)
For `Error` - probably it make sense just leave default value it as is (although Default option would be also useful for consistency)